developer/openshift/dev_setup.sh

```diff
@@ -238,9 +271,20 @@ install_pipeline_service() {
   "$PROJECT_DIR/operator/images/access-setup/content/bin/setup_work_dir.sh" \
     ${DEBUG:+"$DEBUG"} \
     --work-dir "$WORK_DIR" \
-    --kustomization "$GIT_URL/developer/openshift/gitops/argocd?ref=$GIT_REF" |
+    --kustomization "git::$GIT_URL/developer/openshift/gitops/argocd?ref=$GIT_REF" |
```

Comment:
shouldn't we also use PROJECT_DIR here too when `USE_CURRENT_BRANCH` is set ?

Reply:
No. What's happening is that we set everything as usual, but before running the install, we go in and modify the content, which is what's happening line 277.